src/components/general/Search/search.module.scss

```diff
@@ -0,0 +1,50 @@
+$border_color: #BACAD1;
```

Comment:
Get these variables into that variables.scss that I mentioned in coments for Header PR